style.css

```diff
@@ -93,4 +101,212 @@ body {
   .social li {
     flex: 1;
   }
+
+  .work {
+    height: auto;
+    display: grid;
+    grid-template: repeat(3, 1fr) / 32.7rem;
+    justify-content: center;
+    justify-items: center;
+    row-gap: 8.8rem;
+    padding: 1.6rem 1.6rem 0 1.6rem;
+    margin-top: 11.4rem;
+  }
+
+  .project-title {
+    font-weight: 700;
+    font-size: 3.2rem;
+    line-height: 4.4rem;
+    color: #172b4d;
+    margin: 1.2rem 0 0 0;
+  }
+
+  .project {
+    border: 0.1rem solid #dfe1e6;
+    background-color: #fff;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1.8rem 0;
+    padding: 1.6rem 1.6rem 2.8rem 1.6rem;
+    border-radius: 1.6rem;
+  }
+
+  .project-header {
+    flex-basis: 29.5rem;
+  }
+
+  .project-details {
+    flex-basis: 29.5rem;
+  }
+
+  .project-details ul {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    gap: 0 1.2rem;
+    margin: 0;
+    width: 85%;
+    justify-content: center;
+  }
+
+  .project-details ul li {
+    text-align: center;
+  }
+
+  .socialmedia ul li {
+    flex: 1;
+  }
+
+  .project-details ul li:first-child {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #344563;
+    font-weight: 600;
+    flex: 1;
+  }
+
+  .project-details ul li:nth-child(3) {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #7a869a;
+    font-weight: 600;
+    flex: 3;
+  }
+
+  .project-details ul li:nth-child(5) {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #7a869a;
+    font-weight: 600;
+    flex: 1;
+  }
+
+  .project p {
+    flex-basis: 29.5rem;
+    margin: 0;
+  }
```

Comment:
- [ ] Please make sure to stick to the `Figma` design template as much as possible. Your work section description text is too small and doesn't match the `Figma` design. Kindly consider giving it a `font-size` of `15px` as showned in the `Figma` design. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="338" alt="Screenshot 2023-06-02 at 11 58 36" src="https://github.com/AjroudiRida/Portfolio/assets/107192076/0ee0ee20-2832-4ef2-ab3e-29e4e137174e">
</td>
<td>
<img width="318" alt="Screenshot 2023-06-02 at 11 59 15" src="https://github.com/AjroudiRida/Portfolio/assets/107192076/f15b5cf7-c109-4f89-ad4a-07abbd00c7e7">
</td>
</tr>
</table>